src/string-utils/index.ts

```diff
@@ -183,3 +183,8 @@ export const parseText = <
 
   return items;
 };
+
+export const allowLocalUrls = (url: string) => {
+  if (url === "/") return true;
+  return /^\/\w./.test(url);
```

Comment:
I think this technically works for this purpose, but did you mean for the pattern to only test the first character after the slash?

```javascript
> '/apps'.match(/^\/\w./)
['/ap', index: 0, input: '/apps', groups: undefined]
```

I think you may have wanted something more like this (which would also catch the root URL case, and support fragments if we wanted)?

```suggestion
return /^\/(.*)/.test(url);
```

Reply:
Good catch!  I actually just wanted `\w+`

Reply:
@neurosnap I think using `\w+` there would break for paths with multiple slashes, e.g. `/apps/69875/services`, as well as anything that includes a query string, like `/environments?search=asdf`, right?

If you want to support those as well, we can either make the regex more complex, or just simplify this to check that the given string starts with a `/`.

Reply:
I just want to make sure it starts with `/` and a character because I want to negate `//github.com` or w/e.  I added more test cases and they pass just fine as-is.